src/app/portlets/content-types/common/content-types-form/content-types-form.component.ts

```diff
@@ -77,11 +83,29 @@ export class ContentTypesFormComponent extends BaseComponent {
     ngOnInit(): void {
         this.initWorkflowtFieldOptions();
 
+        this.activatedRoute.params.subscribe((params: Params) => {
+            this.contentTypeId = params['id'];
```

Comment:
I don't think we need this here... If I'm correctly it's not being use anywhere, also, we talk about move all the editing things to the "edit" component and just pass things to the form, we don't want the form doing "editing things" in create and stuff.

In fact I think we don't need the Router here at all.